### DISCUSSION TITLE
How to set proxy for cross origin problems?

### DISCUSSION THREAD

**Discussion by Gu-Miao** [parcel-bundler/parcel#4854] [General] [answered]
In react, i cound write a `setupProxy.js` or add `proxy` in `package.json`. But how could i use this with parcel's devServer?

**Comment by DeMoorJasper** [comment, marked as answer]
In Parcel 1 this does not exist, in Parcel 2 it works pretty much the same, the proxy field in package.json is supported and the js file is called `.proxyrc.js`

**Reply by Gu-Miao** [reply]
OK, thank you for the answer, i will try later when 2 is released. 😀

**Reply by DeMoorJasper** [reply]
You can try the beta if you want, it should be fairly stable as long as you don't need any custom plugins
